### PR TITLE
fix dates in wp table displayed in german when english language setting selected

### PR DESCRIPTION
**Pull request by 0xF013**
[`* `#17069 ` fix dates in wp table displayed in german when english language setting selected`](http://community.openproject.org/work_packages/17069)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://0xf013-hotfix-17069-dates-in-wp-table-displayed-in-german-when-.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: app/assets/javascripts/angular/openproject-app.js [hunk 1]
```diff
@@ -166,13 +166,15 @@ openprojectApp
     '$rootScope',
     '$window',
     'flags',
-    function($http, $rootScope, $window, flags) {
+    'TimezoneService'
```
Comment (myabc) [inline app/assets/javascripts/angular/openproject-app.js]: @0xF013 missing comma

Comment (myabc) [inline app/assets/javascripts/angular/openproject-app.js]: there's no :dog2: on this branch

File: app/assets/javascripts/angular/openproject-app.js [hunk 2]
```diff
@@ -166,13 +166,15 @@ openprojectApp
     '$rootScope',
     '$window',
     'flags',
-    function($http, $rootScope, $window, flags) {
+    'TimezoneService',
```
Comment (0xF013) [inline app/assets/javascripts/angular/openproject-app.js:169]: @myabc push -f covers all the tracks :sunglasses:

Comment (myabc) [inline app/assets/javascripts/angular/openproject-app.js:169]: @0xF013 Travis knows :)

**Comment by myabc** [comment]
@0xF013 don't love `initialize` as a function name, but can't think of a better suggestion…

**Comment by 0xF013** [comment]
@myabc 'setupLocale' maybe?

**Comment by myabc** [comment]
@0xF013 yeah, better. renamed in d6708a0e.

**Pull request merged**